_health-care/_js/components/financial-assessment/AnnualIncomeSection.jsx

```diff
@@ -148,4 +150,21 @@ class AnnualIncomeSection extends React.Component {
   }
 }
 
-export default AnnualIncomeSection;
+function mapStateToProps(state) {
+  return {
+    data: state.annualIncome,
+    receivesVaPension: state.vaInformation.receivesVaPension
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+    onStateChange: (field, update) => {
+      dispatch(updateField(['annualIncome', field], update));
+    }
+  };
+}
+
+// TODO(awong): Remove the pure: false once we start using ImmutableJS.
+export default connect(mapStateToProps, mapDispatchToProps, undefined, { pure: false })(AnnualIncomeSection);
```

Comment:
The `undefined` is to prevent the default for `mergeProps`, `Object.assign({}, ownProps, stateProps, dispatchProps)`, from being assigned? Is there a reason we want to not use the default?